packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -34,21 +37,17 @@ import {
   FormatDialect,
 } from "../util/sql";
 import { MetricRegressionAdjustmentStatus } from "../../types/report";
+import { formatInformationSchema } from "../util/informationSchemas";
 
 export default abstract class SqlIntegration
   implements SourceIntegrationInterface {
   settings: DataSourceSettings;
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // @ts-ignore
-  datasource: string;
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // @ts-ignore
-  organization: string;
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // @ts-ignore
-  decryptionError: boolean;
+  datasource!: string;
+  organization!: string;
+  decryptionError!: boolean;
   // eslint-disable-next-line
   params: any;
+  type!: DataSourceType;
```

Comment:
@jdorn I changed this from `string` to `DataSourceType` to be a bit more specific. Oddly, when I change the type of `type` within `types/Integration.ts` from `string` to `DataSourceType` the Google Analytics class throws a type error (but the Mixpanel doesn't). Any suggestions?